### PR TITLE
feat : completed documentation for Alerts page.

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
completed documentation for Alerts page.

### CODE REVIEW

File: Docs/Components-pages/Alerts-page.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                <li class="component"><a href="/components/Inputs/input.html"> Inputs </a></li>
+                <li class="component"><a href="/components/Navigation/Navigation.html"> Navigation </a></li>
+                <li class="component"><a href="/components/responsive-imgs/responsive-imgs.html"> Responsive Images</a></li>
+                <li class="component"><a href="/components/text-utilities/text-utilities.html"> Typography </a></li>
+            </ul>
+        </aside>
+        <section class="content-section">
+            <section class="content-subsection flex">
+                <p class="heading"> Alerts </p>
+                <p class="para-heading">   </p>
```
Comment (vidhi-thakur) [inline Docs/Components-pages/Alerts-page.html:52]: empty `p` tag, could be removed if not necessary

**Review by vidhi-thakur — APPROVED** [review]
added one minor suggestion

File: Docs/Components-pages/Alerts-page.html (hunk 1 above)
Comment (shuklahimanshu81) [reply Docs/Components-pages/Alerts-page.html:52]: Thanks for pointing, will remove that.

**Review by stutisk — APPROVED** [review]
Looks good to merge from my side!

**Pull request merged**